### ISSUE TITLE
Make SVI Slider Yellow/Orange When Engaged; Add SVI Value to Alt Text

### ISSUE THREAD

**Issue by aarontitus** [CrisisCleanup/crisiscleanup-3-web#1112]
**Describe the problem**
The SVI slider settings are saved in user.states. When you log back in, it may be unclear whether the SVI slider is engaged (e.g. less than 100), and therefore whether you are seeing everything on the screen.
Also, it's hard to see the absolute SVI value.

**Describe the solution**
Make the slider button yellow whenever it is engaged (e.g. less than "Everyone"). That way we can ask the person, "Is your SVI slider yellow?" and very quickly ascertain if that's the problem.
Also, the alt text for the SVI slider should be the SVI value, rounded to either 0.0, or 0.00.

**Additional context and instructions**

**Comment by tabiodun** [comment]
I just realized the issue with the second ask is that there is no value associated with the svi level, it just removes a percentage of the cases. What I guess it could show is the lowest or higest svi represented by that point (or both) like a range

**Comment by tabiodun** [comment]
![image](https://user-images.githubusercontent.com/1372010/111707602-ba3cb500-881a-11eb-8b9d-bfa2a1b039ee.png)
